src/pages/page-2.js

```diff
@@ -3,13 +3,160 @@ import { Link } from "gatsby"
 
 import Layout from "../components/layout"
 import Seo from "../components/seo"
+import imgLogo from "../images/Luma.png"
+import imgCart from "../images/Cart.png"
 
 const SecondPage = () => (
   <Layout>
     <Seo title="Page two" />
-    <h1>Hi from the second page</h1>
-    <p>Welcome to page 2</p>
+    {/* <h1>Hi from the second page</h1>
+    <p>Welcome to page 2</p> */}
     <Link to="/">Go back to the homepage</Link>
+    <h1></h1>
+
+    <div className="flex flex-row">
+      <div className="divHeaderCell w-8/12">
```

Comment:
Spend some time here working on the header. Instead of forcing the widths think about `justify-content: center` or `justify-content: space-around`